.lintr

```diff
@@ -0,0 +1,6 @@
+linters: linters_with_defaults(
+    line_length_linter(120),
+    commented_code_linter = NULL,
+    object_usage_linter = NULL
```

Comment:
This last one is useful, but there is a bug in the way lintr works where it views internal functions and references to them as undeclared globals. I have too many internal functions to ignore each line, so disabling this for now.  We could circumvent this by installing the package before linting, but that would requiring installing seurat and hdf5r which takes forever to install.

See [here](https://github.com/r-lib/lintr/issues/27) for more information